.infra/infra/personas.tf

```diff
@@ -0,0 +1,228 @@
+// RAITO PERSONAS
+resource "snowflake_user" "benjamin" {
+  name              = "BenjaminStewart"
+  email             = "b_stewart@raito.io"
+  display_name      = "Benjamin Stewart"
+  first_name        = "Benjamin"
+  last_name         = "Stewart"
+  comment           = "Raito - Access Manager"
+  password          = random_password.persona_password.result
+  default_warehouse = snowflake_warehouse.warehouse.id
+}
+
+resource "snowflake_user" "nick" {
+  name              = "NickNguyen"
+  email             = "n_nguyen@raito.io"
+  display_name      = "Nick Nguyen"
+  first_name        = "Nick"
+  last_name         = "Nguyen"
+  comment           = "Raito - Admin"
+  password          = random_password.persona_password.result
+  default_warehouse = snowflake_warehouse.warehouse.id
+}
+
+resource "snowflake_user" "carla" {
+  name              = "CarlaHarris"
+  email             = "c_harris@raito.io"
+  display_name      = "Carla Harris"
+  first_name        = "Carla"
+  last_name         = "Harris"
+  comment           = "Raito - Observer"
+  password          = random_password.persona_password.result
+  default_warehouse = snowflake_warehouse.warehouse.id
+}
+
+resource "snowflake_user" "dustin" {
+  name              = "DustinHayden"
+  email             = "d_hayden@raito.io"
+  display_name      = "Dustin Hayden"
+  first_name        = "Dustin"
+  last_name         = "Hayden"
+  comment           = "Raito - Owner"
+  password          = random_password.persona_password.result
+  default_warehouse = snowflake_warehouse.warehouse.id
+}
+
+resource "snowflake_user" "mary" {
```

Comment:
If we also want to use this for demo data and app.raito.dev:
I would propose to also add a couple of the other dummy users that we have, coming from Okta. As with the okta sync those users will have tags.

Reply:
We can easily define the SCIM integration. Of course would make it harder for non Raito-developers to setup our test infrastructure

Reply:
Indeed, that's why maybe just a few of those is enough instead of the SCIM. 
SCIM will also be harder for some others probably. + This also gives us the opportunity to add some users on one data source and some others on other data sources (to later test the 'you don't haven an account' stuff :) )